packages/react/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import clsx from 'clsx';
+import {FC, MutableRefObject, ReactElement, useCallback, useEffect, useRef, useState} from 'react';
+import {WithWrapperProps} from '../../models';
+import {baseTheme} from '../../theme/default-theme';
+import {composeComponentDisplayName} from '../../utils';
+import Box from '../Box';
+import './stepper.scss';
+
+export interface StepperProps {
```

Comment:
Maybe we can extend the root component here so `className` could be inherited.
```suggestion
export interface StepperProps extends HTMLAttributes<HTMLDivElement> {
```